openedx/features/student_certificates/views.py

```diff
@@ -117,8 +119,24 @@ def student_certificates(request):
             'certificate_url': "%s%s" % (settings.LMS_ROOT_URL, certificate_url),
             'course_start': start_date.strftime('%b %d, %Y') if start_date else None,
             'completion_date': completion_date.strftime('%b %d, %Y') if completion_date else None,
+            'is_program_cert': False,
         })
 
+    program_credentials = get_credentials(request.user, credential_type='program')
+    for credential in program_credentials:
+        program_name = [attribute['value'] for attribute in credential.get('attributes')
+                        if attribute['name'] == 'program_name']
+        if program_name:
+            user_certificates.append({
+                'course_name': program_name[0],
+                'course_title': program_name[0],
+                'social_sharing_urls': {},
+                'certificate_url': credential.get('certificate_url'),
```

Comment:
shouldn't we have a default value if using `.get()`?

Reply:
I just realized this had to be an empty string for now until the view program certificate page gets ready.

Reply:
if credential has no certificate url then please don't add certificate to list